editor/editor_node.cpp

```diff
@@ -851,8 +851,8 @@ void EditorNode::_resources_changed(const Vector<String> &p_resources) {
 		if (res.is_null()) {
 			continue;
 		}
-
-		if (!res->editor_can_reload_from_file()) {
+		Ref<Script> is_script = res;
+		if (!res->editor_can_reload_from_file() && is_script == nullptr) {
```

Comment:
```suggestion
		if (is_script == nullptr && !res->editor_can_reload_from_file()) {
```
Cheaper condition first

Reply:
btw it should be `is_script.is_null()`.